Source/WarpX.H

```diff
@@ -643,7 +643,9 @@ public:
     void FillBoundaryG   (int lev, amrex::IntVect ng, const bool nodal_sync = false);
     void FillBoundaryAux (int lev, amrex::IntVect ng);
 
-    void SyncCurrent ();
+    void SyncCurrent (
```

Comment:
Just a note: please add doxygen for the new params.

Reply:
Yes, thank you. Done in https://github.com/ECP-WarpX/WarpX/pull/3277/commits/de8942733bc9821681631a27f5f8846992a6af0f.